packages/babel-generator/src/printer.js

```diff
@@ -245,8 +245,8 @@ export default class Printer {
     for (i = 0; i < str.length && str[i] === " "; i++) continue;
     if (i === str.length) return;
 
-    const cha = str[i];
-    const chaPost = str[i + 1];
+    const cha = str[i++];
```

Comment:
Ah but we had a `if (i === str.length) return;` above this that wasn't doing anything

Reply:
As per chat discussion: This doesn't guard against `i+1 === str.length`. The `i===str.length` check guards the `str[i]` access. I'd recommend to refactor this later. But I didn't want to change the semantics, since I don't know anything about this code. :-)

Reply:
imho skipping `++` here and doing:
```
const chaPost = ++i === str.length ? "" : str[i]
```
would be more readable as the increasing of the `i` and its usage would be more collocated in the code

Reply:
Changed it to avoid the unnecessary read of `str[i+1]` unless `cha` is `"/"`.